lib/vector-config/src/schema.rs

```diff
@@ -151,13 +151,16 @@ pub fn generate_number_schema<N>() -> SchemaObject
 where
     N: Configurable + ConfigurableNumber,
 {
+    // TODO: Once `schemars` has proper integer support, we should allow specifying min/max bounds
```

Comment:
Is there a tracking issue or similar we can link here?

Reply:
I'll add an issue for that, good call.

Reply:
Added: https://github.com/vectordotdev/vector/issues/15271